.github/workflows/release-final.yml

```diff
@@ -143,7 +143,7 @@ jobs:
               ref: "main",
               workflow_id: "release-desktop.yml",
               inputs: {
-                branch: ${{ inputs.ref }}
+                branch: "${{ inputs.ref }}"
```

Comment:
any reason ?

Reply:
Yes - [the failure in the workflow](https://github.com/LedgerHQ/ledger-live/actions/runs/12299306845/job/34324784252#step:23:25) was caused by interpreting the value spliced in here (`main`) as a variable name, not a string. Inserting this inside quote marks coerces it to string, and that's why this fixes the problem

Reply:
does't it work before?

Reply:
is it a regression or due to environment change?

Reply:
it was a regression, a mistake introduced by me in [this PR](https://github.com/LedgerHQ/ledger-live/pull/8342)

Reply:
clear ! thanks